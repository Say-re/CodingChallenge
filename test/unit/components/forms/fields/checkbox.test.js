import React from 'react';
import { shallow } from 'enzyme';
import CheckboxComponent from '../../../../../src/components/forms/fields/checkbox'

describe('Checkbox Component Tests', () => {
  test('CheckboxComponent Snapshot Test', async () => {
    const wrapper = shallow(<CheckboxComponent />)
    await expect(wrapper).toMatchSnapshot()
  })
});