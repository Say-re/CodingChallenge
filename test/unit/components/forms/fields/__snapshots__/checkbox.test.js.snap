// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox Component Tests CheckboxComponent Snapshot Test 1`] = `
<div
  className="blockFormFields"
>
  <input
    name="userTest"
    onChange={[Function]}
    type="checkbox"
  />
  <label
    for="test"
  >
    I acknowledge that this is only a test.
  </label>
</div>
`;
